services/data_deletion/clean_deleted_data_from_db.py

```diff
@@ -8,7 +8,7 @@ def clean_deleted_data(self):
         This method deletes data that has been marked for
         deletion from the database
         """
-        print("cleaning data...")
+        ("cleaning data...")
```

Comment:
Why remove the `print`?

Reply:
Ohh sorry, this change was not meant to happen. I must have removed it while cleaning up my print statements. Let me put it back